### PR TITLE
[CI] Use scoop instead of choco for installing tools on Windows

### PR DESCRIPTION
**Pull request by AnotherFoxGuy**
Apparently chocolatey sometimes suffers from an issue where it is unable to connect to the choco servers
This PR fixes that by completely replacing choco with [scoop](https://scoop.sh/)

(see https://github.com/tenacityteam/tenacity/pull/545#discussion_r697915564)
<details>
<summary>Checklist</summary>

- [x] I have signed off my commits using `-s` or `Signed-off-by`\* (See: [Contributing § DCO](https://github.com/tenacityteam/tenacity/blob/master/CONTRIBUTING.md#developer-certificate-of-origin))
- [x] I made sure the code compiles on my machine
- [x] I made sure there are no unnecessary changes in the code\*
- [x] I made sure the title of the PR reflects the core meaning of the issue you are solving\*
- [x] I made sure the commit message(s) contain a description and answer the question "Why do those changes fix that particular issue?" or "Why are those changes really necessary as improvements?"\*

\* indicates required

</details>

### CODE REVIEW

File: .github/workflows/cmake_build.yml [hunk 1]
```diff
@@ -83,14 +83,15 @@ jobs:
 
       AUDACITY_ARCH_LABEL: ${{ matrix.config.arch }}
 
+      SCOOP: C:\Scoop\
```
Comment (Be-ing) [inline .github/workflows/cmake_build.yml]: Why is this needed? It doesn't seem to be used?

Comment (AnotherFoxGuy) [reply .github/workflows/cmake_build.yml]: That environment variable is to specify the scoop install location

Comment (Be-ing) [reply .github/workflows/cmake_build.yml]: What command uses it? `iwr -useb get.scoop.sh | iex`?

Comment (AnotherFoxGuy) [reply .github/workflows/cmake_build.yml]: Both `iwr -useb get.scoop.sh | iex` and `scoop install sccache` uses it

File: .github/workflows/cmake_build.yml [hunk 2]
```diff
@@ -109,9 +110,18 @@ jobs:
         JFROG_ARTIFACTORY_NUGET_TOKEN: ${{secrets.JFROG_ARTIFACTORY_NUGET_TOKEN}}
 
     - name: Install dependencies
+      if: runner.os != 'Windows'
       run: |
         exec bash "scripts/ci/dependencies.sh"
 
+    - name: "[Windows] Install dependencies"
+      if: runner.os == 'Windows'
+      shell: pwsh
```
Comment (Be-ing) [inline .github/workflows/cmake_build.yml:142]:
```suggestion
      shell: pwsh
      env:
        # Install path of scoop
        SCOOP: C:\Scoop\
```
Please set the environment variable only where it is needed instead of globally for all steps.

**Review by Be-ing — DISMISSED** [review]

**Review by caughtquick — DISMISSED** [review]

**Comment by emabrey** [comment]
Please move the Windows install step back into the bash script or move all the components of the bash script into the github workflow (Mac and Linux).

**Review by emabrey — DISMISSED** [review]

**Review by Be-ing — DISMISSED** [review]

**Comment by AnotherFoxGuy** [comment]
Flatpak died because one of the servers was down
CodeQL died because it is still using the deps.sh script, I will fix that soon
